source/gui/addonGui.py

```diff
@@ -157,15 +157,17 @@ def __init__(self, parent):
 		AddonsDialog._instance = weakref.ref(self)
 		# Translators: The title of the Addons Dialog
 		title = _("Add-ons Manager")
-		wx.Dialog.__init__(self, parent, title=title)
+		# Translators: The title of the Addons Dialog when add-ons are globally disabled
+		titleWhenAddonsAreDisabled = _("Add-ons Manager (add-ons globally disabled)")
+		wx.Dialog.__init__(self, parent, title=title if not globalVars.appArgs.disableAddons else titleWhenAddonsAreDisabled)

```

Comment:
Can you split this across multiple lines please?

```suggestion
		wx.Dialog.__init__(
			self,
			parent,
			title=title if not globalVars.appArgs.disableAddons else titleWhenAddonsAreDisabled
		)
```